aws-py-voting-app/__main__.py

```diff
@@ -0,0 +1,298 @@
+# Copyright 2016-2020, Pulumi Corporation.  All rights reserved.
+
+import json
+import base64
+import pulumi
+import pulumi_aws as aws
+import pulumi_docker as docker
+
+# Get the password to use for Redis from the pulumi config
+config = pulumi.Config()
+redis_password = config.require("redis-password")
+redis_port = 6379
+
+# The ECS cluster in which our application and databse will run
+app_cluster = aws.ecs.Cluster("app-cluster")
+
+# Creating a VPC and a public subnet
+app_vpc = aws.ec2.Vpc("app-vpc",
+    cidr_block="172.31.0.0/16",
```

Comment:
Do you want to hardcode this? Or make it configurable and then set a default if not set?

Reply:
I think that unlike other configurations, the CIDR block might not be as interesting for users to change. Since this is a private network, 172.31.0.0 should always work for every project without creating deployment conflicts (even if you deploy multiple stacks of this), and if users really want to change it, I think it's obvious for them where to do it.